src/rlang/node.c

```diff
@@ -30,7 +30,9 @@ r_obj* r_new_pairlist(const struct r_pair* args,
 
 // Shallow copy of a node tree
 r_obj* r_node_tree_clone(r_obj* x) {
```

Comment:
I didn't end up using this, but the generalisation to both calls and pairlists makes sense.